### PR TITLE
Back to top : Add CSS variables

### PR DESCRIPTION
**Pull request by louismaximepiton**

### CODE REVIEW

**Review by MewenLeHo — APPROVED** [review]
Looks good 👌

**Comment by julien-deramond** [comment]
@louismaximepiton there's a conflict to resolve here after the recent merge of a back-to-top fix.

File: site/layouts/partials/footer.html [hunk 1]
```diff
@@ -55,7 +55,7 @@ <h5>Community</h5>
 </footer>
 <!-- Boosted mod: back-to-top component -->
 <nav aria-label="Back to top" class="back-to-top">
-  <a href="#top" class="back-to-top-link btn btn-icon btn-secondary" data-o-label="Back to top">
+  <a href="#top" class="back-to-top-link btn btn-icon btn-secondary" data-bs-label="Back to top">
```
Comment (julien-deramond) [inline site/layouts/partials/footer.html:58]: `data-o-label` → `data-bs-label` should be in the migration guide

File: site/content/docs/5.1/components/back-to-top.md [hunk 2]
```diff
@@ -95,9 +95,21 @@ Use a `title` attribute instead of `data-o-label` to ensure a visible label is s
...
 
-## Sass
+## CSS
 
-### Options
+### Variables
+
+{{< added-in "5.2.0" >}}
+
+As part of Boosted's evolving CSS variables approach, back-to-top now use local CSS variables on `.back-to-top` for enhanced real-time customization. Values for the CSS variables are set via Sass, so Sass customization is still supported, too.
```
Comment (julien-deramond) [inline site/content/docs/5.1/components/back-to-top.md:104]:
```suggestion
As part of Boosted's evolving CSS variables approach, Back to tops now use local CSS variables on `.back-to-top` for enhanced real-time customization. Values for the CSS variables are set via Sass, so Sass customization is still supported, too.
```

**Review by julien-deramond — CHANGES_REQUESTED** [review]
Just few comments in order to improve the docs. Otherwise LGTM!

**Review by julien-deramond — APPROVED** [review]
LGTM!

**Pull request merged**